Versions/Common/logic.mounts.lua

```diff
@@ -52,4 +52,24 @@ function BeStride:CheckLoanedMount()
 	end
 	
 	return nil
+end
+
+function BeStride:CanUseTargetsMount()
+	if (not self:DBGet("settings.mount.copytargetmount") or not BeStride:IsMainline()) then return false end
+	if (self:MovementCheck() or not UnitExists("target")) then return false end
+
+	local mountId = nil
+	AuraUtil.ForEachAura("target", "HELPFUL", 40, function(...)
```

Comment:
It doesn't look like `AuraUtil.ForEachAura` is defined in WotLK so I may have to switch to manually looping through `UnitBuff("target",index)` for that (or both).